doc/source/theory/fluid_dynamics/fluid_dynamics.rst

```diff
@@ -12,3 +12,4 @@ Fluid dynamics
     stabilization
     linear_solvers
     rheology
+    vof
```

Comment:
I wonder if this is the right place to add the description of VOF. Don't you think that we should maybe add a multiphysics tab in the theory section that contains this. Just for it to be uniform, e.g., with the examples subsections?

Reply:
I agree wiht @lpsaavedra